libraries/core/Cargo.toml

```diff
@@ -34,3 +35,4 @@ url = { version = "2.5.4", optional = true }
 git2 = { workspace = true, optional = true }
 fs_extra = "1.3.0"
 splitty = "1.0.2"
+zenoh = { workspace = true, optional = true }
```

Comment:
Could be good to put it somewhere else than core because every time we modify core, including dependency we add breaking changes and cannot make backward compatibility as it is tied to dora messages

Reply:
I think there might be a bigger issue that is to separate dora-core and dora-messages @phil-opp